modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-custom-scheduler.adoc
+
+[id="nodes-custom-scheduler-deploying_{context}"]
+= Deploying a custom scheduler
+
+To include a custom scheduler in your cluster, include the image for a custom scheduler in a deployment.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have a scheduler binary.
++
+[NOTE]
+====
+Information on how to create a scheduler binary is outside the scope of this document. For an example, see link:https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers[Configure Multiple Schedulers] in the Kubernetes documentation. Note that the actual functionality of your custom scheduler is not supported by Red Hat.
```

Comment:
@sferich888 @reestr Can you also review this statement?

Reply:
+1 

However we may want to point users at https://docs.openshift.com/container-platform/4.7/nodes/scheduling/nodes-scheduler-default.html and https://docs.openshift.com/container-platform/4.7/nodes/scheduling/nodes-scheduler-default.html#nodes-scheduler-default-modifying_nodes-scheduler-default to explain how and in what ways they can configure the OCP scheduler in a supported way.